editors/sc-ide/widgets/help_browser.cpp

```diff
@@ -120,6 +120,18 @@ HelpBrowser::HelpBrowser( QWidget * parent ):
     setFocusProxy(mWebView);
 }
 
+void HelpBrowser::setServerPort(int serverPort)
+{
+    mServerPort = serverPort;
+}
```

Comment:
in the future you can define things like this inline in the class definition, since the implementation is trivial and one-line:

```cpp
bool helpBrowserHasFocus() const;
void setServerPort(int serverPort) { mServerPort = serverPort; }
```

i'd rather have that than 5 lines in the cpp file